x-pack/plugins/observability/public/components/shared/core_web_vitals/index.tsx

```diff
@@ -90,6 +90,7 @@ export function CoreVitals({
             loading={loading}
             thresholds={CoreVitalsThresholds.LCP}
             helpLabel={LCP_HELP_LABEL}
+            hasVitals={Boolean(coreVitalPages)} // can coreVitalPages ever been negative? Should I use coreVitalPages > 0 instead?
```

Comment:
I mean to remove this comment, but it does provide context for why I did this for anyone reviewing, while it remains here at least. Updating now....